src/app/debug/table/table.component.ts

```diff
@@ -582,4 +589,31 @@ export class TableComponent implements OnInit, OnDestroy {
       this.reportsInProgressThreshold = time;
     });
   }
+
+  handleFilterErrorContext(): string {
+    let result: string = '';
+    let moreThanOne: boolean = false;
+    for (const [key, value] of this.filterErrorDetails) {
+      let typeLabel: string = key;
+      switch (typeLabel) {
+        case 'int': {
+          typeLabel = 'number';
+          break;
+        }
+        case 'long': {
+          typeLabel = 'decimal number';
+          break;
+        }
+        case 'timestamp': {
+          typeLabel = 'date time';
+          break;
+        }
+      }
```

Comment:
I can add it, but string is currently always counted as correct input so it wouldn't ever be displayed. I'll add it in case I missed some scenario where it would give an error